Tools/autotest/sim_vehicle.py

```diff
@@ -678,6 +676,10 @@ def start_vehicle(binary, opts, stuff, spawns=None):
         c = ["-I" + str(i)]
         if spawns is not None:
             c.extend(["--home", spawns[i]])
+        if opts.sysid is None:
+            c.extend(["--sysid", str(i)])
```

Comment:
do we end up with sysid 0 here?

Reply:
Yes, incrementing by 1 puts it in the correct range. I guess there should also be a check if it is <= 255.